tests/api/test_topic_entity_tag.py

```diff
@@ -224,9 +229,12 @@ def test_validation(self, test_topic_entity_tag, test_reference, test_mod, auth_
             response = client.get(f"/topic_entity_tag/{test_topic_entity_tag.new_tet_id}")
             assert response.json()["validation_by_author"] == "validated_wrong"
 
-    @pytest.mark.webtest
     def test_validation_wrong(self, test_topic_entity_tag, test_reference, test_mod, auth_headers, db):  # noqa
-        with TestClient(app) as client:
+        with TestClient(app) as client, \
+                patch("agr_literature_service.api.crud.topic_entity_tag_crud.get_ancestors") as mock_get_ancestors, \
+                patch("agr_literature_service.api.crud.topic_entity_tag_crud.get_descendants") as mock_get_descendants:
+            mock_get_ancestors.return_value = []
```

Comment:
In this case, the mock happens to prevent the web lookup, but the values aren't needed so mock_get_... are empty, and that's okay ?

Reply:
Yes

Reply:
Cool, thanks